website/docs/docs/deploy/cloud-ci-job.md

```diff
@@ -24,17 +24,17 @@ If you previously configured your dbt project by providing a generic git URL tha
 
 When you set up a dbt Cloud CI job with a [webhook configured](#configuring-a-webhook), dbt Cloud will listen for webhooks from GitHub, GitLab, or Azure DevOps indicating that a new pull request has been opened or updated with new commits. When one of these webhooks is received, dbt Cloud will enqueue a new run of the CI job. 
 
-Crucially, this run will build into a temporary schema using the prefix `dbt_cloud_pr_`. This schema isolation acts as a quasi-staging environment, so that you can see the builds resulting from the code associated with the PR's commit. The unique schema name can be found in the run details for the given run, as shown in the following image:
+Once you open a pull request, dbt Cloud builds the models affected by the code change in a temporary schema using the prefix `dbt_cloud_pr_`. dbt Cloud then  runs the tests for these models as a check. This process provides a staging environment where you can check the run status and builds resulting from the code associated with the pull request's commit. When the CI job completes, you can see the run status directly in the pull request. The CI job enables you to deploy new code to production with confidence. The unique schema name, your project ID, and pull request ID (`dbt_cloud_pr_1862_917`) can be found in the run details for the given run as shown in the following image:
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/using_ci_dbt_cloud.png" title="Viewing the temporary schema name for a run triggered by a PR"/>
 
 After completing the dbt run, dbt Cloud will update the pull request in GitHub, GitLab, or Azure DevOps with a status message indicating the results of the run. The status message will state whether the models and tests ran successfully or not. You can enable a setting in your git provider that makes "successful pull request checks" a requirement to merge code. And finally, once the pull request is closed or merged, dbt Cloud will delete the temporary schema from your <Term id="data-warehouse" />.
 
 dbt Cloud might not drop the temporary schema from your data warehouse if your project has database / schema customization using the [`generate_database_name`](/docs/build/custom-databases#generate_database_name) / [`generate_schema_name`](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros. For more information, refer to [Temp PR schema limitations](/docs/deploy/cloud-ci-job#temp-pr-schema-limitations).
 
-### Configuring a webhook
+### Configure a job trigger
```

Comment:
@joellabes I tried to clarify that this is not the newly released webhooks, but a job trigger as seen here (the tab is actually also called "webhook"): 
![Screenshot 2023-02-02 at 1 03 59 PM](https://user-images.githubusercontent.com/3880403/216449811-9b5ad5a0-6a07-4dab-b630-b5ef4da0ce63.png)

Reply:
I've [asked in the webhooks channel](https://dbt-labs.slack.com/archives/C04KR8M25AM/p1675375172516919) if we should rebrand this page

Reply:
This new UI is now part of this PR thanks to @thorn14